src/figuresClasses.ts

```diff
@@ -1,19 +1,122 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  a?: number;
+  b?: number;
+  c?: number;
+  r?: number;
+  width?: number;
+  height?: number;
+  getArea: () => number;
 }
 
 export class Triangle {
```

Comment:
all figures must implement class Figure

Reply:
not fixed 
![image](https://github.com/mate-academy/ts_figures_classes/assets/96265844/a915fc29-60e7-4d4d-bdbb-36cdb8f62a39)